### DISCUSSION TITLE
key press to take you to different areas of your patch

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#374] [Feature Requests] [answered]
I'm working on a patch and I'm constantly moving around between 3 areas of it and I thought....

wouldn't it be nice if you could press a key and it would take you to that area.

I'm already using your area feature to highlight bits of code but if i could assign each area to a number for example you could quickly jump around a patch.

Anyway, just a thought, would anyone else find this useful?

**Comment by steam0r** [comment, marked as answer]
hey,

something really close to what you want is the bookmark feature for ops. click this little flag thingie:

![Bildschirmfoto 2022-10-17 um 12 04 00](https://user-images.githubusercontent.com/151271/196150603-9ad517a5-3f66-4f80-a2eb-727adfb8724e.png)

and you will see your bookmarked ops when clicking on an empty space on the patchpanel:

![Bildschirmfoto 2022-10-17 um 12 04 07](https://user-images.githubusercontent.com/151271/196150647-ac65b64c-83e8-4617-999e-08e2b5b20865.png)

also works with areas, of course. hope that helps!

**Reply by marioepsley** [reply]
oooo that's cool

**Reply by steam0r** [reply]
try the new things in ctrl-p "command palette" on dev.cables.gl, you can directly jump to subpatches with only your keyboard for example :)
